### DISCUSSION TITLE
Flipper Fox Finder

### DISCUSSION THREAD

**Discussion by EDU-MDAnderson** [K3YOMI/Wall-of-Flippers#28] [Ideas]
Would it be possible to create a flipper version to fox hunt other flippers performing BLE attacks?

**Comment by k3yomi** [comment]
Sadly the Flipper Zero's BLE stack is pretty basic and is limited to certain functionality.  With that said, the only way I can think of at this time is using an ESP32 that supports BLE to perform actions similar to Wall of Flippers.
